tests/benchmark/query/execution.cpp

```diff
@@ -416,6 +416,61 @@ BENCHMARK_TEMPLATE(Aggregate, MonotonicBufferResource)
 
 BENCHMARK_TEMPLATE(Aggregate, PoolResource)->Ranges({{4, 1U << 7U}, {512, 1U << 13U}})->Unit(benchmark::kMicrosecond);
 
+template <class TMemory>
```

Comment:
Do you know where the results of these benchmarks can be found?

Reply:
Honestly, I thought they were run with benchmarks but now I can't find them.